umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso19115_2/spatial.clj

```diff
@@ -81,17 +92,32 @@
     (into {} (for [[k ^String v] (partition 2 (str/split orbit-string #":? "))]
                [(keyword k) (Double/parseDouble v)]))))
 
+(defn- parse-vertical-domains
+  "Parse the vertical domain from the ISO XML document. Vertical domains are encoded in an ISO XML
+  document as a single string like this:
+  \"Type: Some type Value: Some value\""
+  [doc]
+  (when-let [vertical-domains (select doc vertical-string-xpath)]
+    (for [vertical-domain vertical-domains]
```

Comment:
So I tried changing the above to the following:
(for [vertical-domain vertical-domains
        :let [vertical-domains (select vdoc vertical-string-xpath)]
        :when (vertical-domains)]
but I get an unable to resolve this symbol vertical-domains error.  What am I doing wrong?